### ISSUE TITLE
plugin not working anymore because bs has implementet catchup

### ISSUE THREAD

**Issue by annahana** [BuXXe/movian-bs-plugin#4]
The plugin is at the moment just usable to show what bs has as series.
All videos say that file not found . The reason is that bs.to has implemented anti bot catchup to see how is human an how is pc.

**Comment by BuXXe** [comment]
Yeah thanks for the post. I already saw that in the forum and they integrated it because their database was crawled and many uploads were taken down. I dont think that I will be working on a captcha solver but perhaps, if i get in the mood, there could be a way to integrate it in the plugin somehow... stay tuned

**Comment by LFriede** [comment]
It is possible to show the captcha in a webbrowser popup with movian. This worked for me on the Linux version what i use for testing but crashes movian on PS3 (tested on Firmware 3.55).

**Comment by BuXXe** [comment]
yeah i worked on a similar method. could u share your code? i will have a look

**Comment by LFriede** [comment]
Sure, but it is not very much, it was just a test and i stopped trying as i noticed that it's not working on PS3.

```javascript
    var getHosterLink = showtime.httpGet("http://bs.to/"+episodeLink);
    var dom = html.parse(getHosterLink.toString());
    var directlink = dom.root.getElementByClassName('hoster-player')[0].attributes.getNamedItem("href").value.toString();

    // If the link is pointing to the redirect page we have to solve a captcha
    if (directlink.indexOf('https://bs.to/out/') > -1) {
      var redirect = require('native/popup').webpopup(directlink, 'Please solve the captcha', 'https://syndication.exoclick.com/ads-iframe-display.php');
      if (redirect.result == 'trapped') {
        directlink = decodeURIComponent(redirect.args.p);
      }
    }
```

That changes around line 50 in bs.js do the trick for videos hosted on **vivo**, other hosters require other trigger urls.

**Comment by BuXXe** [comment]
thx perhaps we / i  can get to something useful

**Comment by BuXXe** [comment]
I wrote a mail to your gmx address. hope you still use it ;)

**Comment by LFriede** [comment]
I replied.

**Comment by BuXXe** [comment]
fixed in 0.6 release. Took some time but finally
